modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -63,11 +65,13 @@ namespace
 
         static auto ocv_pkg =
 #if !defined(GAPI_STANDALONE)
-            combine(cv::gapi::core::cpu::kernels(),
-                    cv::gapi::imgproc::cpu::kernels());
+            combine(combine(cv::gapi::core::cpu::kernels(),
```

Comment:
I think it's better than adding render kernel to opencv kernel package

Reply:
Can you please write a FIXME on adding N-arg version of combine? Opening an internal issue would be nice as well

Reply:
Added FIXME